content/millennium/r4/medications/medication-request.md

```diff
@@ -133,12 +144,33 @@ We are currently always returning a value of "Order" in the intent field. We rec
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/56770371
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/56770371 
 
 #### Response
 
 <%= headers status: 200 %>
 <%= json(:R4_MEDICATION_REQUEST_ENTRY) %>
+<%= disclaimer %>	
+
+#### Patient Authorization Request For Active Status	
+
+    GET https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/MedicationRequest/262448077 
```

Comment:
These URLs should be of C1941 domain
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Reply:
comment addressed